projects/demo/src/modules/components/calendar/index.html

```diff
@@ -22,78 +22,38 @@
         </p>
 
         <tui-doc-example
-            id="base"
-            heading="Basic"
-            [content]="example1"
+            *ngFor="let example of examples; let index = index"
```

Comment:
`let index = index + 1` why not

and then

```html
[component]="index | tuiComponent"
[content]="index | tuiExample"
```

```
[ngSwitch]="index"
```

Reply:
agree

Reply:
Sorry, but i can't add it, because i will get `Parsing error: unexpected token +, expected identifier, keyword or column`